docs/builtin_workflows.rst

```diff
@@ -160,7 +164,7 @@ Additionally, a number of anisotropy scalar images are produced such as QA, GFA
 ========================
 
 This workflow implements DSI Studio's q-space diffeomorphic reconstruction (QSDR), the MNI space
-(ICBM-152) version of GQI, followed by automatic fiber tracking (autotrack) [Yeh2020]_ [Yeh2022]_
+(ICBM-152) version of GQI, followed by automatic fiber tracking (autotrack) :footcite:p:`autotrack,yeh2022population`
```

Comment:
When I render this locally there is no comma. Is that normal?

Reply:
it ends up looking like this: 
<img width="191" alt="image" src="https://github.com/user-attachments/assets/0ca7964f-1d64-429d-9737-dabaa13e28d0">
which isn't a dealbreaker but is a little odd

Reply:
Ah, I see what you mean. I think we might need to write our own [`bibtex_foot_reference_style`](https://sphinxcontrib-bibtex.readthedocs.io/en/latest/usage.html#referencing-style) to change that, unfortunately.

Reply:
not worth it IMO